.github/workflows/build-check-install.yaml

```diff
@@ -200,3 +200,19 @@ jobs:
       - name: Install R package
         run: R CMD INSTALL ${{ env.PKGBUILD }}
         shell: bash
+
+      - name: System tests
+        run: |
+          print("Creating dependency table")
+          x <- staged.dependencies::dependency_table("openpharma/stageddeps.food", project_type = "repo@host", feature = "fixgarden@main", verbose = 1)
```

Comment:
- feature should be replaced with `ref` and should also test with a tag once that has been added to example repos
- also can remove verbose = 1 as that is now the default throughout